.github/workflows/ci.yml

```diff
@@ -19,8 +19,13 @@ jobs:
           - "3.10"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
+      - name: Harden Runner
+        uses: step-security/harden-runner@248ae51c2e8cc9622ecf50685c8bf7150c6e8813
+        with:
+          egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
```

Comment:
Could we drop these new actions for the time being? We'd love to get the scorecard and pinned actions merged, but I think we need to have some internal discussion before adding additional instrumentation to each workflow.